macros/src/main/scala/gestalt/macros/DefMacros.scala

```diff
@@ -2,30 +2,34 @@ import scala.gestalt._
 
 object plusObject {
   def apply(a: Int, b: Int): Int = meta {
+    import toolbox._
```

Comment:
Currently we haven't done the path-dependent path refactoring, this import is mandatory to import the implicit conversions.